### PR TITLE
[docs] Images release tag links

### PR DESCRIPTION
**Pull request by IvyBazan**
## Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

*Description:*
Added links to release tags for available images

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: available_images.md [hunk 1]
```diff
@@ -82,17 +82,19 @@ You can pin your version by adding the version tag to your URL as follows:
...
-|TensorFlow 2.2.0   |inference               |No 	                           |GPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.2.0-gpu-py37-cu102-ubuntu18.04  |v1.0           |
-|TensorFlow 2.2.0   |inference               |No 	                           |CPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-inference:2.2.0-cpu-py37-ubuntu18.04        |v1.0           |
+|[TensorFlow 2.2.0](https://github.com/aws/deep-learning-containers/releases/tag/v2.0-tf-2.2.0-tr-py37-gpu)   |training 	             |Yes                        	   |GPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.2.0-gpu-py37-cu102-ubuntu18.04   |v2.0           |
+|[TensorFlow 2.2.0](https://github.com/aws/deep-learning-containers/releases/tag/v1.1-tf-2.2.0-tr-py37)   |training 	             |Yes                        	   |GPU 		| 3.7 (py37) 			    |763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.2.0-gpu-py37-cu101-ubuntu18.04   |v1.1           |
```
Comment (TusharKanekiDey) [inline available_images.md:89]: We have also released v1.2 images as well now. Should we include in new PR.

Comment (IvyBazan) [reply available_images.md:89]: Sounds good to me.

Comment (nskool) [reply available_images.md:89]: @IvyBazan While linking to github releases is a good idea, the links may not always point to the latest images. We should probably hold this PR/close it till we have a better idea to link to the latest images. Till then, the image URIs that we already have should suffice.

Comment (nskool) [reply available_images.md:89]: @IvyBazan Closing this PR for above reasons. Thanks for creating the PR!

**Pull request closed without merging**